.github/workflows/on-release-branch-commit.yml

```diff
@@ -28,7 +28,9 @@ jobs:
         run: |
           set +e
           fail=0
-          for i in deploy-testnet.yml deploy-mainnet.yml; do
```

Comment:
Let's diff all the workflows

Reply:
This has been changed and the workflow is going to check all action and workflow .yml files.